### ISSUE TITLE
Craniotabes

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11496]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: [39677085] Craniotabes is characterized by the softening of skull bones in newborns. It can be associated with
conditions like rickets, congenital syphilis, and osteogenesis imperfecta. In otherwise healthy newborns,
craniotabes is often linked to in utero vitamin D deficiency.

**Parent term (use hpo.jax.org/app)**
Abnormal calvaria morphology  HP:0002683

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Osteogenesis imperfecta ORPHA:666
Hypophosphatasia ORPHA:436

**Your nano-attribution (ORCID)**
0009-0005-6714-5727
